Vagrantfile

```diff
@@ -0,0 +1,66 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Use this Vagrantfile to run p2p-tests on an Intel macOS host.
+# This requires VirtualBox to be installed.
+#
+#   brew install vagrant
+#   vagrant up
+#   vagrant ssh
+#     cd /vagrant
+#     cargo build
+#     cd p2p-tests
+#     sudo -E ./contributor-fork-replication-1-test.ts
+
+Vagrant.configure("2") do |config|
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 8192
+    v.cpus = 4
+  end
+
+  config.vm.box = "ubuntu/focal64"
+  config.vm.provision "shell", inline: <<~SHELL
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get -y update
+    apt-get -y upgrade
+    apt-get -y --no-install-recommends install \
+      build-essential \
+      ca-certificates \
+      cmake \
+      curl \
+      git \
+      jq \
+      procps; \
+    rm -rf /var/lib/apt/lists/*;
```

Comment:
I don’t think this is necessary. It saves a bit of space but makes subsequent apt commands slower.

Reply:
65e982ac431e796be804a71bc79cd2b51fe5e6fc